app/src/controllers/objectPermission.js

```diff
@@ -31,6 +31,20 @@ const controller = {
         permCode: utils.mixedQueryToArray(req.query.permCode)
       });
       const response = utils.groupByObject('objectId', 'permissions', result);
+
+      if (utils.isTruthy(req.query.bucketPerms)) {
+        const objectIds = await objectPermissionService.getObjectIdsWithBucket(userId, bucketIds);
```

Comment:
note: the `bucketIds` param is used to optionally scope the object permission search to one(or more) buckets, if undefined (not passed as an input param) the search is for object perms across all buckets.

To include objects with implicit (bucket-level permission) in the response, by passing in `bucketPerms=true`, i think it's good to pass the `bucketIds` param on to the `getObjectIdsWithBucket()` function.